lib/cleanFile.js

```diff
@@ -13,6 +13,7 @@
 
 import chalk from 'chalk';
 import { fs } from 'style-dictionary/fs';
+import { logVerbosityLevels } from './buildInEnums.js';
```

Comment:
```suggestion
import { logVerbosityLevels } from './buildInEnums.js';

const { silent } = logVerbosityLevels;
```

Reply:
Let's see, for what kind of API we also decide for `const { scss } = transformGroups;` and my concerns of duplicate values between different enums.

Reply:
Addressed similarly to overlapping imports w local variables:
`const { scss: scssGroup } = transformGroups;`